tests/features/frontend_cache_purge.feature

```diff
@@ -510,3 +512,19 @@ Feature:
     And I fill in "Moderation notes" with "Immediately publishing this"
     And I press "Save"
     Then an informational message is logged with type "nexteuropa_varnish" and a message matching "Clearing paths: /more-basic-pages, /, /even-more-basic-pages"
+
+  Scenario: No purge are instructed if a part of the configuration has disappeared.
+    Given the following cache purge rules:
+      | Content Type | Paths to Purge         |
+      | page         | /more-basic-pages, /   |
```

Comment:
One rule is enough for this test, with the path "frontend-cache-purge-publish-immediately"